.circleci/config.yml

```diff
@@ -34,8 +23,37 @@ jobs:
           paths:
             - node_modules
             - app/node_modules
-          key: v1-dependencies-{{ checksum "yarn.lock" }}-{{ checksum "app/yarn.lock"}}
+            - ../.cache/Cypress
+          key: dependencies-{{ checksum "yarn.lock" }}-{{ checksum "app/yarn.lock"}}
 
       # run tests!
       - run: yarn eslint
       - run: yarn test-chrome-headless
+      - run:
+          name: Build the app so that server can be started
+          command: cd app && yarn build && cd ../
+      - run:
+          name: Start server with correct env for cypress tests
+          command: >
+            yarn generate-cert && ADALITE_ENABLE_HTTPS=true
+            ADALITE_SERVER_URL='https://localhost:3000' PORT=3000
+            ADALITE_BLOCKCHAIN_EXPLORER_URL='https://explorer-testnet.adalite.io'
```

Comment:
I'd consider creating a custom .env.test file in the cypress folder and explicitly loading it into yarn dev. For example in my local setup some environment variables were set differently (ADALITE_TREZOR_CONNECT_URL) and cypress was crashing - took me a while to debug

A way to achieve it that I thing should work would be in `loadConfig.js` replacing

```javascript
require('dotenv').config()

```

with

```
require('dotenv').config({ path: process.env.DOTENV_CONFIG_PATH})
```

which then would allow for

`DOTENV_CONFIG_PATH=./my/path/to/.env.anything` where the proper env variables for cypress could be set, without any interference from local .env file